css-four-column-layout/styles.css

```diff
@@ -0,0 +1,44 @@
+.container {
+  width: 1400px;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: "Barlow Condensed", sans-serif;
+  font-family: "Changa One", cursive;
+  font-family: "Delicious Handrawn", cursive;
+  font-family: Roboto, sans-serif;
+}
+
+img {
+  width: 100%;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.header {
+  text-align: center;
+  font-size: 1.5rem;
+}
+
+.column-fourth-two > img {
+  width: 20%;
+  padding-left: 130px;
```

Comment:
You shouldn't need to be setting such high `px` values. If you need to do something this large, then you are probably doing something "hacky" to get this to work.